Source/Logger/trace.cpp

```diff
@@ -46,6 +50,35 @@ int vsprintf_s(char(&buffer)[SIZE], _Printf_format_string_ char const* format, v
 #endif
 #endif
 
+#if HC_PLATFORM_IS_MICROSOFT
+TRACELOGGING_DEFINE_PROVIDER(
+    g_hTraceLoggingProvider,
+    "libHttpClient",
+    (0x2b6274b2, 0x159f, 0x4e6f, 0xb8, 0x0, 0xd, 0x17, 0x0, 0x9e, 0x14, 0x7a) // {2B6274B2-159F-4E6F-B800-0D17009E147A}
```

Comment:
This is the same existing GUID PF currently uses since that should already be registered in the wherever-that-is. Unless the name there needs to match what's defined here, I assume nothing needs to be changed service-side.

Reply:
That work hasn't been done yet anyhow AFAIK so I don't think that's a concern